packages/qwik/src/core/tests/use-on.spec.tsx

```diff
@@ -647,4 +649,76 @@ describe.each([
     await trigger(document.body, 'div', 'click');
     await expect(document.querySelector('div')).toMatchDOM(<div>1</div>);
   });
+
+  it('issue 7230, when multiple useOnWindow are used in a component that is not rendered, it should add multiple script nodes', async () => {
```

Comment:
please rename it to:
`#7230 - when multiple useOn are used in a component that is not rendered, it should add multiple script nodes`